### PR TITLE
Make debugging easier by only listening on a special queue

### PR DESCRIPTION
**Pull request by chriskuehl**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,8 +21,13 @@ the celery worker. To automatically source and unsource the virtualenv when
...
+To start the celery worker, run `python -m create.worker --debug`. The celery
+worker will by default use the config file already on supernova, but you can
+specify your own config file to use for development with the `-c` or `--config`
+parameter to either one. More help is available with `-h` or `--help`. To exit
+the virtualenv when you are done working on create, just type `deactivate`.
+
+When running with `--debug`, your celery worker will only listen on a queue
+named after the hostname it runs on. This means that your worker won't serve
+real tasks (e.g. users using prod ocfweb), and that you can always hit your
+worker by specifying the right queue (e.g. `supernova`).
```
Comment (jvperrin) [inline README.md:34]: Maybe explain how to actually hit the queue named after the hostname?

**Review by jvperrin — APPROVED** [review]
Nice, not sending the emails and not taking tasks from the production queue are pretty great

**Comment by abizer** [comment]
you may want to fix this git history with a rebase

**Comment by kkuehlz** [comment]
lol rip ckuehl's commit

**Comment by abizer** [comment]
h

**Comment by wporr** [comment]
Jesus finally

I merged his commit instead of rebasing and it caused issues

**Review by kkuehlz — APPROVED** [review]

**Pull request merged**